src/components/Searchbar.jsx

```diff
@@ -25,7 +25,7 @@ const Searchbar = () => {
       inputProps={{
         classNames: {
           input: 'ml-1',
-          inputWrapper: 'h-[45px]'
+          inputWrapper: 'bg-white h-[45px] w-[1000px]'
```

Comment:
1. setting a width like this is not mobile responsive. If you want to align with with navbar, you should add px-[1.5rem]
2. I didn't explicitly set bg-white because for dark mode. Does the searchbar looks black in light mode? I think I made the change on my pr and it's not merged yet. Try to remove `bg-white` and add `dark:dark` in classname in App.js like this ` <div className="App dark:dark">` . Then it will be white in light mode, and black in dark mode.

Reply:
Aligning searchbar width with the navbar will be fixed if you copy and paste the codes that I commented on below.

Reply:
Used `bg-white` and add `dark:bg-black-800` method because `<div className="App dark:dark">` wasn't handling light mode